lib/byron/src/Cardano/Wallet/Byron/Compatibility.hs

```diff
@@ -136,24 +141,28 @@ type NodeVersionData =
 -- Chain Parameters
 
 
-mainnetBlockchainParameters :: W.BlockchainParameters
-mainnetBlockchainParameters = W.BlockchainParameters
-    { getGenesisBlockHash = W.Hash $ unsafeFromHex
-        "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb"
-    , getGenesisBlockDate =
-        W.StartTime $ posixSecondsToUTCTime 1506203091
-    , getFeePolicy =
-        W.LinearFee (Quantity 155381) (Quantity 43.946) (Quantity 0)
-    , getSlotLength =
-        W.SlotLength 20
-    , getEpochLength =
-        W.EpochLength 21600
-    , getTxMaxSize =
-        Quantity 8192
-    , getEpochStability =
-        Quantity 2160
-    , getActiveSlotCoefficient =
-        W.ActiveSlotCoefficient 1.0
+mainnetBlockchainParameters :: W.GenesisBlockParameters
+mainnetBlockchainParameters = W.GenesisBlockParameters
+    { staticParameters = W.BlockchainParameters
+        { getGenesisBlockHash = W.Hash $ unsafeFromHex
+            "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb"
+        , getGenesisBlockDate =
+            W.StartTime $ posixSecondsToUTCTime 1506203091
+        , getSlotLength =
+            W.SlotLength 20
+        , getEpochLength =
+            W.EpochLength 21600
+        , getEpochStability =
+            Quantity 2160
+        , getActiveSlotCoefficient =
+            W.ActiveSlotCoefficient 1.0
+        }
+    , txParameters = W.TxParameters
+        { getFeePolicy =
+            W.LinearFee (Quantity 155381) (Quantity 43.946) (Quantity 0)
+        , getTxMaxSize =
+            Quantity 8192
```

Comment:
To be double-checked. I am pretty sure that we put there the _current_ tx max size, but I am not sure it was the genesis one. I know it used to be 4096 bytes _at some point_.

Reply:
So, I just checked and indeed:

```json
        "maxTxSize": "4096",
```

Reply:
OK fixed.